compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -371,7 +371,6 @@ pub enum AtomicOrdering {
     NotAtomic = 0,
     Unordered = 1,
     Monotonic = 2,
-    // Consume = 3,  // Not specified yet.
```

Comment:
Why not keep this comment? Makes it easy to know why there's a random hole at a glance